test/adapters/http/read.spec.js

```diff
@@ -470,7 +470,7 @@ describe('HTTP read tests', function() {
                 return check_juttle({
                     program: 'read http -headers { Accept: "' + contentType + '"} ' +
                              '          -url "' + server.url + '/objects?foo=' +
-                             Array(2048).join('X') + '&count=100" | head 1'
+                             Array(8*1024).join('X') + '&count=100" | head 1'
```

Comment:
I'll do my best to explain and then put a comment here for future readers of the code. Some of the CSV parsers read a ahead a small amount 1-2KB and then parse that and fire events, then read again ahead. So when you tell the parser to `end,close,stop` it will in fact spew out a few more entries (as many as can fit in its internal buffer). When I switched the csv parser here the new one must have a larger buffer and therefore we were getting more entries. I made the test lines much longer resulting in filling the internal buffer quicker and therefore not exceeding the expected 50 entries of reading ahead.